### PR TITLE
#80 part of User Story #12. Almost polished front-end (error message …

### PR DESCRIPTION
**Pull request by Golemwardox**
…CSS needs to be fixed). Functioning Back-end with mockaroo. Need to implement unit tests

### CODE REVIEW

File: back-end/app.js [hunk 1]
```diff
@@ -34,4 +35,41 @@ app.post("/room", function (req, res) {
   res.send({ roomId });
 });
 
+
+app.post("/invite/:roomId", async function(req, res) {
```
Comment (Jen-Lopez) [inline back-end/app.js:39]: Just writing a comment here for future reference: 

This should be where we would redirect (`res.redirect("/wait")`) to the waiting room.

**Review by Jen-Lopez — APPROVED** [review]

File: back-end/app.js [hunk 2]
```diff
@@ -34,4 +35,41 @@ app.post("/room", function (req, res) {
...
+    res.send({ result });
+  }
+  catch(error) {
+    res.send(error);
+  }
+
+})
+
 app.listen(8000);
+// export app;
```
Comment (Jen-Lopez) [inline back-end/app.js:75]: btw, not sure what the purpose of this `export app` is. I tested it out and it's not really necessary.

Comment (Golemwardox) [reply back-end/app.js:75]: I don't think it is necessary either!

File: front-end/src/pages/JoinRoom.jsx [hunk 3]
```diff
@@ -4,23 +4,45 @@ import React from 'react';
 import InviteCodeInput from '../components/InviteCodeInput';
 import RoomButton from '../components/RoomButton';
 import Spacer from '../components/Spacer';
-import { Link } from "react-router-dom";
+import { join_post } from "../utils/api";
```
Comment (yunko1803) [inline front-end/src/pages/JoinRoom.jsx:7]: Can you try to use Request.js for join_post method?

**Review by yunko1803 — APPROVED** [review]

**Pull request merged**